scene/main/viewport.cpp

```diff
@@ -1790,22 +1792,22 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 			Viewport *embedder = nullptr;
 			Vector2 viewport_pos;
 
-			if (is_embedding_subwindows()) {
-				embedder = this;
-				viewport_pos = mpos;
-			} else {
-				// Not an embedder, but may be a subwindow of an embedder.
-				Window *w = Object::cast_to<Window>(this);
-				if (w) {
-					if (w->is_embedded()) {
-						embedder = w->_get_embedder();
+			//if (is_embedding_subwindows()) {
```

Comment:
Why was the following commented out?

Reply:
I think that i was experimenting at that time!

I forgot about this PR!
This currently works?

Reply:
I updated this PR!